homeassistant/components/harmony/switch.py

```diff
@@ -44,7 +45,7 @@ def name(self):
     @property
     def unique_id(self):
         """Return the unique id."""
-        return f"{self._data.unique_id}-{self._activity}"
+        return f"{self._activity_id}"
```

Comment:
```suggestion
        return f"activity-{self._activity_id}"
```

In case we end up with harmony switches that aren't activities in the future, it might make sense to prefix this now.

Reply:
Good call - I prefixed with `activity_` to keep the migration logic a bit simpler.